space/space.go

```diff
@@ -301,7 +301,8 @@ func (r *GormRepository) listSpaceFromDB(ctx context.Context, q *string, userID
 	}
 
 	// ensure that the result list is always ordered in the same manner
-	db = db.Order("spaces.updated_at DESC")
+	db = db.Order(*sort)
```

Comment:
I dislike that the parameter `sort` is just a string and not it's own type. You could essentially just pass in anything here. Also, what is if `sort` is `nil`? Then you're dereferencing a `nil` value.

Reply:
done, created new type